operators/src/source/gdal_source/mod.rs

```diff
@@ -420,11 +434,48 @@ impl GdalRasterLoader {
         tile_information: TileInformation,
         tile_time: TimeInterval,
     ) -> Result<RasterTile2D<T>> {
-        crate::util::spawn_blocking(move || {
-            Self::load_tile_data(&dataset_params, tile_information, tile_time)
-        })
+        let is_vsi_curl = dataset_params.file_path.starts_with("/vsicurl/");
```

Comment:
What about `/vsicurl_streaming/` or `/vsizip//vsicurl/` ?

Reply:
I could do `file_path.contains("/vsicurl")` but then it would detect a local folder name vsicurl also as being a remote file. So should I add your two examples explicitly or do we need a gdal file path parser that extracts the virtual file systems from a path?

Reply:
a80d47091639d8e20e4c5ef664b1a0c6bd751082